ci-conda.Dockerfile

```
@@ -1,6 +1,6 @@
-ARG CUDA_VER=11.8.0
-ARG LINUX_VER=ubuntu22.04
-ARG PYTHON_VER=3.10
+ARG CUDA_VER=-notset
+ARG LINUX_VER=notset
+ARG PYTHON_VER=-notset
```

I'd drop the extra dashes. It's a bit more confusing than needed for the relatively minor benefit of readable words in errors.
```suggestion
ARG CUDA_VER=notset
ARG LINUX_VER=notset
ARG PYTHON_VER=notset
```

---

Ok sure, I'm indifferent 🤷🏻 

You picked up on exactly why I had them... so you get `cuda-notset-*` instead of `cudanotset`.